src/vtkIECTransformLogic.cxx

```diff
@@ -52,6 +52,7 @@ vtkIECTransformLogic::vtkIECTransformLogic()
   this->CoordinateSystemsMap[FlatPanel] = "FlatPanel";
   this->CoordinateSystemsMap[WedgeFilter] = "WedgeFilter";
   this->CoordinateSystemsMap[Patient] = "Patient";
+  this->CoordinateSystemsMap[PatientImagePlane] = "PatientImagePlane";
```

Comment:
This transform only works for CT scans with regular slice spacing, which is not mandatory. In fact, sometimes (not too often) the patients are scanned with slices of 1mm separation in the area of interest and then 5mm (these cases we will skip and focus on the more easy cases, otherwise the computation becomes difficult and slow).
Thus we should give a name that clarifies this, and specify that also in the docu.
In other words, this is supposed to work only with "regular grids" and not with "rectilinear grids".
Consequently, I would suggest naming it "PatientRegularGrid" or sth like that.

Reply:
see https://en.wikipedia.org/wiki/Regular_grid